docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -774,32 +774,34 @@ test('My Test', async t => {
 
 CSS selectors passed to the [Selector](../../reference/test-api/selector/constructor.md) constructor cannot identify elements in the shadow DOM.
 
-To select a shadow element, initialize a selector with client-side code and use the [shadowRoot](https://developer.mozilla.org/en-US/docs/Web/API/Element/shadowRoot) property to get and return the element from the shadow DOM.
+To access and traverse the shadow DOM attached to an element, identify the element with a `Selector` and use the [Selector.shadowRoot](../../reference/test-api/selector/shadowroot.md) method to
+target the [shadow root](https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot) of the element. Use other `Selector` methods to traverse the shadow tree from there.
+
+> Important! You can not target an element returned by `shadowRoot()` or use it in assertions.
```

Comment:
1) Do you mean *you cannot perform an action with an element returned by shadowRoot()*? Because `target an element` is not specific enough.
2) Are you sure it should be a warning box, not a regular one? Will there be any grave consequences or hard-to-identify errors if a user doesn't follow the note? Here and in the reference topic.

Reply:
1. You cannot perform actions, assertions or anything for that matter because `shadowRoot` method returns a [fragment node](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment) and not an element.

> You can not perform actions with a node returned by `shadowRoot()` or use it in assertions.

2. It errors out. I guess it can be a regular note since the error is straightforward:

>1) The specified selector is expected to match a DOM element, but it matches a document fragment node.